cmd/move_config.go

```diff
@@ -40,7 +40,8 @@ func newMoveConfigCmd(out io.Writer) *cobra.Command {
 	}
 
 	flags := cmd.Flags()
-	settings.AddBaseFlags(flags)
+	settings.AddFlags(flags)
```

Comment:
I don't think this command should have all the flags as it doesn't communicate with the kubernetes cluster.

Did you mean to change this line?